beautyjoy/BJC-Teacher-Tracker
# CS169 Iteration 1 Pull Request

### [Pivotal Tracker Link][tracker]


[tracker]: https://www.pivotaltracker.com/story/show/your-story-id

We decided to include the links to each of the pull requests that were made over the course of this iteration. See below. PRs are listed in chronological order, from earliest to most recent:

Change reason to denial reason: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/29](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/29)

Add to field: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/30](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/30)

Add email resend flash message: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/31](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/31)

Add request more info feature as admin: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/32](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/32)

Edit redirect to use current path: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/33](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/33)

Fix email template reseed problem: [https://github.com/cs169/BJC-Teacher-Tracker-App/pull/34](https://github.com/cs169/BJC-Teacher-Tracker-App/pull/34)


## What this PR does:


Implements iteration 1 features, including request more information button, the "to" field in the email template, and the flash message on resending emails. We also changed "reason" to "denial_reason" and fixed the duplicate email entry problem that occurs when you run "rails db:seed" after running migrations.

### Include screenshots, videos, etc.

#### Who authored this PR?


Everyone.

### How should this PR be tested?

See cucumber and RSpec tests, and view the latest deployment at [https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard](https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard).

* Is there a deploy we can view?
[https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard](https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard)
* What do the specs/features test?
To make sure everything works as expected
* Are there edge cases to watch out for?
No

#### Are there any complications to deploying this?

No



Run "rails db:migrate" and "rails db:seed".

### Checklist:
